README.md

```diff
@@ -1,24 +1,314 @@
+
+## Objectives
+
+1. Identify CSS syntax
+2. Identify CSS use formats
+3. Utilize various types of CSS selectors
+4. Utilize various types of color values in CSS
+5. Recognize and implement CSS comments
+
+## Identify CSS Syntax
+
+Before we turn to the various ways we can extend CSS rules, let's go over the
+foundational syntax CSS uses.
+
+<img src="https://curriculum-content.s3.amazonaws.com/fewds/css-syntax.png" alt="CSS syntax diagram labeling selector, property and value" width="500" />
+
+We create a CSS rule by defining the selector, which matches the HTML element we
+want to style. Inside the curly braces we declare the property we want to change
+and, after the colon, we set the value we want to change that property to. We
+end the declaration with a semicolon. Here, we are selecting the `p` element and
+displaying its color as blue.
+
+## Identify CSS Use Formats
+
+There are three distinct formats that you might see CSS appear in: inline,
+internal (or embedded) and external.
+
+Inline includes the styles directly into the HTML element with the `style`
+attribute.
+
+```
+<p style="color: blue;"></p>
+```
+
+While you might see this sort of styling in something like the code for an HTML
+email, this is generally not best practice for two reasons. The primary reason
+is because it only affects that single element. If we want all paragraph
+elements on our page to appear blue, we would have to add that attribute to
+every element individually, which is inefficient and difficult to maintain into
+the future. That brings us to the second reason to avoid inline CSS: it breaks
+our principle of separation of content and presentation.
+
+Internal CSS is inside of `style` tags in the HTML document's `head` section.
+
+```
+<!DOCTYPE html>
+<html>
+  <head>
+    <style>
+      p { color: blue; }
+    </style>
+  </head>
+  <body>
+
+  <p>This is a paragraph.</p>
+
+  </body>
+</html>
+```
+
+This rule will display all paragraphs in this document as blue, which is a step
+up in scope from the inline styles that only apply to single elements. But this
+CSS will only apply to the single document. Other paragraph elements on other
+pages in the same website will be unaffected.
+
+If we want our CSS to carry across various pages, we can use an external
+stylesheet. This is a separate CSS file that we link in the `head` of HTML
+documents.
+
+```
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+
+  <p>This is a paragraph.</p>
+
+  </body>
+</html>
+```
+
+With the `link` tag, we can use the relation attribute to define the target as a
+stylesheet and the link source our CSS file that contains all the CSS we want to
+use on our site. This is by far the easiest way to link CSS to HTML and apply
+styles across all of our pages.
+
+## Utilize Various Types of CSS Selectors
+
+CSS gives us a wide range of ways that we can select elements on the page. Some
```

Comment:
I love your writing style. It's like my own. That means it's perfect.

But, I think you could say.

> CSS has many ways to select elements on the page.